index.js

```diff
@@ -11,7 +11,7 @@ if (!databaseUri) {
 }
 
 var api = new ParseServer({
-  databaseURI: databaseUri || 'mongodb://localhost:27017/dev',
+  databaseURI: process.env.DATABASE_URI || 'mongodb://localhost:27017/dev',
```

Comment:
see line 7.  it is coalesced from the environment variables.  added in #8 I believe.